linux-headers/tests/test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+#/ Usage: test.sh <pkg_ident>
+#/
+#/ Example: test.sh core/php/7.2.8/20181108151533
+#/
+
+set -euo pipefail
+
+if [[ -z "${1:-}" ]]; then
+  grep '^#/' < "${0}" | cut -c4-
+	exit 1
+fi
+
+
+echo "This package provides the headers for the linux kernel and has no tests"
```

Comment:
Is this necessary if we're not having tests? Seems like bloat.

Reply:
Do you mean specifically this `echo` statement, or the entire `test.sh` file?
The file itself adds value because it records that testing has been considered for this plan; and CI will be able to find and execute this script and consider the plan build to be successful. 
The echo statement seems useful because the author is documenting why the plan can be signed-off for testing with no actual tests. I guess it also gives us assurance that there's a 0-returning statement at the end of the file.